public/locales/pseudo-LOCALE/grafana.json

```diff
@@ -574,4 +574,4 @@
       "option-tooltip": "Cľęäř şęľęčŧįőŉş"
     }
   }
-}
```

Comment:
🤔 idk why this is here

Reply:
@joshhunt i see your name in the blame 😉 halp!

Reply:
Was yarn i18n:extract ran manually? (I can’t see a reason why it should have been). When was this branched from main (we merged that touched this file today?

You’re safe to just not commit this in your PR, and I can double check later that our lint/tests doesn’t make this file dirty in main at the moment.